infra/auth-server/src/models/authtoken-blacklist.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+
+module.exports = (sequelize, DataTypes) => {
+  const AuthTokenBlacklist = sequelize.define(
+    'AuthTokenBlacklist',
+    {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER
+      },
+      ethAddress: {
```

Comment:
Do we need to store any of this additional data? It would seem like the only thing we need is the `authToken` column, and a reason column might be nice.

Reply:
Yep, just the authToken and another status/reason column would be enough. Will update it.